pandas/types/__init__.py

```diff
@@ -0,0 +1,8 @@
+import warnings
+
+
+warnings.warn(
+    "The pandas.types module is deprecated and will be removed in a future "
+    "version. Please import from the pandas.core.dtypes module instead.",
```

Comment:
Here as well, the "Please import from the pandas.core.dtypes module instead." should be removed.

There is part of these functions available in `pandas.api.types`, but not everything. But maybe we can point to that in general?